scripts/run-package-task.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const runProcess = require('./tasks/run-process');
+
+const readFileAsync = util.promisify(fs.readFile);

```

Comment:
util is not required anymore. you can use native fs.promises.readFile

Reply:
Depends on the version of Node you're running on.